src/anim/controller/anim-controller.js

```diff
@@ -420,6 +420,9 @@ class AnimController {
     }
 
     removeNodeAnimations(nodeName) {
+        if ([ANIM_STATE_START, ANIM_STATE_END, ANIM_STATE_ANY].includes(nodeName)) {
```

Comment:
it's likely this
`[ANIM_STATE_START, ANIM_STATE_END, ANIM_STATE_ANY]`
allocates array each time the function runs. Maybe should be stored in some const somewhere.

Reply:
Actually, I think there's another problem here. `Array#includes` is not supported in IE11. And remember, Babel just transpiles language features. I don't think it polyfills standard library functions like this.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes
I would maybe just swap for:
```javascript
    if (nodeName === ANIM_STATE_START ||
        nodeName === ANIM_STATE_END ||
        nodeName ===  ANIM_STATE_ANY) {
```

Reply:
Sure, i've set this as a constant now.